spec/models/child_spec.rb

```diff
@@ -56,6 +56,7 @@
 
```

Comment:
Along with the validation specs above (line 13 - 48) can we add a validation spec for the inclusion validator, for example:
```
expect(subject).to validate_inclusion_of(:active).in_array(%w[Y N])
```

Also could you duplicate the date_param `it` block for `last_active_date`